src/theory/strings/theory_strings.cpp

```diff
@@ -143,7 +143,11 @@ TheoryStrings::TheoryStrings(context::Context* c, context::UserContext* u,
   d_true = NodeManager::currentNM()->mkConst( true );
   d_false = NodeManager::currentNM()->mkConst( false );
 
-  d_card_size = 128;
+  d_card_size = 256;
```

Comment:
Nit: could just write this as `d_card_size = options::stdPrintASCII() ? 256 : 128;`.